paperqa/prompts.py

```diff
@@ -23,7 +23,7 @@
     '"I cannot answer."'
     "For each part of your answer, indicate which sources most support "
     "it via citation keys at the end of sentences, "
-    "like (Example2012Example pages 3-4). Only cite from the context "
+    "like {example_citation}. Only cite from the context "
```

Comment:
Can you now rename the variable name `qa_prompt` to `qa_prompt_template`, since that's what it is now

Reply:
Think prompt and template are synonyms in our code base